docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -481,7 +481,7 @@ Flush metadata cache
 .. _delta-lake-write-support:
 
 Updating data
-"""""""""""""
+^^^^^^^^^^^^^
```

Comment:
![Screenshot from 2023-06-13 11-25-52](https://github.com/trinodb/trino/assets/57260417/097e7bf4-aa7c-4dbb-8833-737a5b4eb769)

New appearance of the _SQL support_ section

compared to https://trino.io/docs/current/connector/delta-lake.html#sql-support

Reply:
![Screenshot from 2023-06-13 14-32-17](https://github.com/trinodb/trino/assets/57260417/3f318408-04b7-4921-9a63-4874fee13d10)